### PR TITLE
fix: path import for stories

### PR DESCRIPTION
**Pull request by cmejet**
This PR fixes the import path for the injection to stories/index.ts. It also adds a new line in the index.ts for the component generation to meet the prettier format standards

## Changes

- export.ejs new line added to meet prettier standards
- inject_story.ejs updates the import path 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

Fixes # issue #44

### CODE REVIEW

File: _templates/react-native-typescript-component/new/inject_story.ejs [hunk 1]
```diff
@@ -3,5 +3,6 @@ inject: true
 to: <%= root %>/storybook/stories/index.ts
 append: true
 ---
+
 <% formattedName = h.inflection.transform(name, ['undasherize', 'classify']) -%>
-import '../../<%= root %>/<%= path %>/<%= formattedName %>/<%= formattedName %>.stories';
\ No newline at end of file
+import '../../<%= path %>/<%= formattedName %>/<%= formattedName %>.stories';
```
Comment (jsjoeio) [inline _templates/react-native-typescript-component/new/inject_story.ejs]: Woo! 💯

File: _templates/react-native-typescript-component/new/export.ejs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
 to: <%= root %>/<%= path %>/<%= h.inflection.transform(name, ['undasherize', 'classify']) %>/index.ts
 ---
-export * from './<%= h.inflection.transform(name, ['undasherize', 'classify']) %>';
\ No newline at end of file
+export * from './<%= h.inflection.transform(name, ['undasherize', 'classify'])
```
Comment (jsjoeio) [inline _templates/react-native-typescript-component/new/export.ejs]:
```suggestion
export * from './<%= h.inflection.transform(name, ['undasherize', 'classify']) %>'
```
Guessing your linter/prettier did this but this should stay on the same line

File: _templates/react-native-typescript-component/new/export.ejs [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
+%>';
```
Comment (jsjoeio) [inline _templates/react-native-typescript-component/new/export.ejs]:
```suggestion
%>';
```

File: _templates/react-native-typescript-component/new/export.ejs (hunk 2 above)
Comment (cmejet) [reply _templates/react-native-typescript-component/new/export.ejs]: Yes, thanks linter...I'll fix

**Comment by jsjoeio** [comment]
@all-contributors please add @cmejet for code

**Pull request merged**